SwiftUIBasics/Views/PlansView.swift

```diff
@@ -1,18 +1,100 @@
 //
-//  PlansView.swift
-//  SwiftUIBasics
+//  ContentView.swift
+//  ExercisesUI
 //
-//  Created by Diplomado on 09/12/23.
+//  Created by Diplomado  on 09/12/23.
 //
 
 import SwiftUI
 
-struct PlansView: View {
+struct ContentView2: View {
     var body: some View {
-        Text(/*@START_MENU_TOKEN@*/"Hello, World!"/*@END_MENU_TOKEN@*/)
+        VStack(alignment:.center){
+            Text("Choose")
+                .font(.system(size: 32, weight: .black, design: .serif))
+                .fontDesign(.rounded)
+            Text("Your Plan")
+                .font(.system(size: 32, weight: .black, design: .serif))
+                .fontDesign(.rounded)
+            HStack {
+                CardView(iconName: "", title: "Basic", description: "Per month", price: "$9")
+                    .foregroundStyle(.white)
+                    .background(.purple)
+                    .cornerRadius(6.0)
+                ZStack {
+                    CardView(iconName: "", title: "Pro", description: "Per month", price: "$19")
+                        .background(.gray)
+                        .cornerRadius(6.0)
+                    BadgeView(text: "Best for designer", padding: 5)
+                        .foregroundStyle(.white)
+                        .background(.orange)
+                        .padding(.top, 200)
+                }
+            }
+            
+            ZStack {
+                CardView(iconName: "sparkles", title: "Team", description:     "                    Per month                   ", price: "299")
```

Comment:
when you uses spaces to center a text you are not having a responsive design.
- what would the equivalent of an space to a screen point?